api/v1/health/openapi.yaml

```diff
@@ -0,0 +1,178 @@
+---
+swagger: '2.0'
+info:
+  title: Cilium-Health API
+  description: Cilium Health Checker
+  version: v1beta
+x-schemes:
+- unix
+basePath: "/v1beta"
+produces:
+- application/json
+consumes:
+- application/json
+paths:
+  "/hello":
+    get:
+      summary: Say hello to cilium-health
+      description: |
+        Returns a successful status code if this cilium-health instance is
+        reachable.
+      responses:
+        '200':
+          description: Success
+  "/healthz":
+    get:
+      summary: Get health of Cilium node
+      description: |
+        Returns health and status information of the local node including
+        load and uptime, as well as the status of related components including
+        the Cilium daemon.
+      responses:
+        '200':
+          description: Success
+          schema:
+            "$ref": "#/definitions/HealthResponse"
+        '500':
+          description: Failed to contact local Cilium daemon
+          x-go-name: Failed
+          schema:
+            "$ref": "../openapi.yaml#/definitions/Error"
+  "/status/probe":
+    put:
+      summary: Get connectivity status of the Cilium cluster
```

Comment:
The summary should already indicate that it is synchronous

`Run synchronous connectivity probe to determine status of Cilium cluster`

Reply:
Ack, will fix.